src/styles/global.scss

```diff
@@ -5,6 +5,16 @@
 
 :root {
   --polaris-version-number: '{{POLARIS_VERSION}}';
+  // This value matches the name of the animation defined below.
+  // The animation name is stored in a custom property so that the scss mixins
+  // in our public api that reference this animation can do so with
+  // `var(--polaris-SkeletonShimmerAnimationName)` instead of
+  // `:global(polaris-SkeletonShimmerAnimation)`. This allows us to expose a
+  // single set of public sass API files instead of needing distinct sass API
+  // entrypoints for consumers that use css-modules in locals mode (such as scss
+  // files in apps using sewing-kit) and for consumers not using css-modules.
+  // stylelint-disable-next-line value-keyword-case
+  --polaris-SkeletonShimmerAnimationName: polaris-SkeletonShimmerAnimation;
```

Comment:
Is there a standard for naming custom properties? (I was under the impression kebab-case was king, but I might be wrong)

Reply:
Yeah kebab-case is what most of the stuff in ThemeProvider does. Have updated the name.

I've kept this prefixed as `--polaris-` (while the ThemeProvider stuff uses `--p-`) to keep variable prefixes consistent within this file